charts/producer-app/templates/job.yaml

```diff
@@ -1,6 +1,6 @@
 {{- $root := . -}}
```

Comment:
I've also renamed this file since `job.yaml` fits both cases. Hope that's okay.

Reply:
Yes, I think this should not cause any problems